uavcan/equipment/air_data/1027.RawAirData.uavcan

```diff
@@ -0,0 +1,29 @@
+#
+# Raw Air Data.
+#
+
+#
+# Heater State
+# 
+uint8 HEATER_UNAVAILABLE    = 0
+uint8 HEATER_WORKING        = 1
+uint8 HEATER_OVERCURRENT    = 2
+uint8 HEATER_OPENCIRCUIT    = 3
+uint8 heater_state
+
+#
+# Pressure Data
+#
+float32 static_pressure                 # Pascal
+float32 differential_pressure           # Pascal
+
+#
+# Temperature Data
+#
+float16 sensor_temperature              # Kelvin
+float16 static_air_temperature          # Kelvin
```

Comment:
Just came to mind that Static air temperature is actually not the raw temperature read by sensor, despite it being called OAT sensor.
This is not really huge matter for regular UAV's low speeds, and can be left as it is with addition of an appropriate comment.